Resources/Prototypes/Corvax/Stack/other_stacks.yml

```
@@ -0,0 +1,7 @@
+- type: stack
+  id: SheetPrinter
+  name: paper
+  icon: { sprite: Objects/Misc/bureaucracy.rsi, state: paper }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)